src/content/en/updates/2017/01/scrolling-intervention.md

```diff
@@ -0,0 +1,150 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: We know that scrolling responsiveness is critical to the user's engagement with a website on mobile, yet touch event listeners often cause serious scrolling performance problems.
+
+{# wf_updated_on: 2017-01-10 #}
+{# wf_published_on: 2017-01-10 #}
+{# wf_tags: intervention,chrome56 #}
+{# wf_featured_image: /web/updates/images/generic/warning.png #}
+{# wf_featured_snippet: We know that scrolling responsiveness is critical to the user's engagement with a website on mobile, yet touch event listeners often cause serious scrolling performance problems. #}
```

Comment:
Just a nit - but this is your hook, this is what shows up on the /web/updates/ landing page (and you can use HTML in here). You might want to make it a bit more exciting. For example:

> Scrolling responsiveness is critical to the user's engagement with a website on mobile, yet touch event listeners often cause serious scrolling performance problems! With `magic` you can avoid all that. Your users will love you and other developers will be envious of your **1337** skills.